app/celery/tasks.py

```diff
@@ -107,21 +109,31 @@ def process_job(job_id):
     ).enumerated_recipients_and_personalisation:
         process_row(row_number, recipient, personalisation, template, job, service)
 
+    job_complete(job, service, template, False, start)
```

Comment:
This would be easier to read if you used named parameters here, as in resumed=True

Reply:
done